### PR TITLE
Introduce Test cases

### PR DESCRIPTION
**Pull request by riseshia**
Related to #19

### CODE REVIEW

File: test/graphs/tasks_test.exs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+defmodule Gannbaruzoi.TasksTest do
+  use Gannbaruzoi.ModelCase
+
+  test "find_or_create_dummy with no registed user" do
```
Comment (marocchino) [inline test/graphs/tasks_test.exs]: It isn't match described things and contain tests.

Comment (riseshia) [reply test/graphs/tasks_test.exs]: Agree, I can't understand my naming, too. XD
I will fix this after #16 merged!

**Comment by riseshia** [comment]
Please review this @marocchino XD

File: test/graphs/tasks_test.exs [hunk 2]
```diff
@@ -0,0 +1,171 @@
+defmodule Gannbaruzoi.TasksTest do
+  use Gannbaruzoi.ModelCase
+
+  def task_with_user! do
+    user = insert!(:user)
+    insert!(:task, user: user)
+  end
+
+  describe "query Tasks" do
+    test "query tasks returns all task" do
```
Comment (marocchino) [inline test/graphs/tasks_test.exs]: duplicated describe:
test "returns all task" do

File: test/graphs/tasks_test.exs [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+        }
+        """
+        |> Absinthe.run(Gannbaruzoi.Schema)
+      expected_keys = ~w/description estimated_size id status type/
+      assert expected_keys == Map.keys(task)
+    end
+  end
+
+  describe "mutation CreateTask" do
+    test "returns new task" do
```
Comment (marocchino) [inline test/graphs/tasks_test.exs]: do not have when part.
"returns new task as valid args."

File: test/graphs/tasks_test.exs [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+          }
+        }
+        """
+        |> Absinthe.run(Gannbaruzoi.Schema)
+      expected_keys = ~w/description estimated_size id status type/
+      assert expected_keys == Map.keys(task)
+    end
+  end
+
+  describe "mutation CreateTask" do
```
Comment (marocchino) [inline test/graphs/tasks_test.exs]: the mutation name is createTask.

File: test/graphs/tasks_test.exs [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+              description
+              estimated_size
+              type
+              status
+            }
+          }
+        }
+        """
+        |> Absinthe.run(Gannbaruzoi.Schema)
+      expected_keys = ~w/description estimated_size id status type/
```
Comment (marocchino) [inline test/graphs/tasks_test.exs]: any reason it isn't `~w(description estimated_size id status type)`?

File: test/graphs/tasks_test.exs [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+          }
+        }
+        """
+        |> Absinthe.run(Gannbaruzoi.Schema)
+      assert 1 == length(errors)
+    end
+  end
+
+  describe "mutation Update Task" do
+    test "updates task" do
```
Comment (marocchino) [inline test/graphs/tasks_test.exs]: same here. when part is missing.

File: test/graphs/tasks_test.exs [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+            }
+          }
+        }
+        """
+        |> Absinthe.run(Gannbaruzoi.Schema)
+      expected_keys = ~w/description estimated_size id status type/
+      assert expected_keys == Map.keys(task)
+    end
+
+    test "fails to update task" do
```
Comment (marocchino) [inline test/graphs/tasks_test.exs]: so on

File: test/graphs/tasks_test.exs [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+          }) {
+            id
+          }
+        }
+        """
+        |> Absinthe.run(Gannbaruzoi.Schema)
+      assert to_string(task.id) == actual_id 
+    end
+
+    test "fails to delete task" do
```
Comment (marocchino) [inline test/graphs/tasks_test.exs]: so on

**Comment by riseshia** [comment]
Todo:
- [x] Split query string to Actual query and Argument
- [x] Dry Absinthe run

**Pull request merged**